pkg/ovssubnet/controller/multitenant/bin/openshift-ovs-multitenant

```diff
@@ -54,19 +54,17 @@ del_ovs_port() {
 add_ovs_flows() {
     ovs_port=$(ovs-ofctl -O OpenFlow13 dump-ports-desc br0  | grep ${veth_host} | cut -d "(" -f 1 | tr -d ' ')
 
-    ovs-ofctl -O OpenFlow13 add-flow br0 "table=3,cookie=0x${ovs_port},priority=100,in_port=${ovs_port},ip,nw_src=${ipaddr},actions=load:${tenant_id}->NXM_NX_REG0[],goto_table:4"
+    ovs-ofctl -O OpenFlow13 add-flow br0 "table=3,priority=100,in_port=${ovs_port},ip,nw_src=${ipaddr},actions=load:${tenant_id}->NXM_NX_REG0[],goto_table:4"
```

Comment:
We could, but (other than consistency with AddOFRules) is there any reason to? A cookie-based-on-IP would not be any more unique than the IP address itself, so deleting-by-cookie wouldn't be any different from deleting-by-IP.